pkg/utils/watchJob.go

```diff
@@ -83,3 +86,27 @@ func JobLogs(experimentName string, engineName string, client *kubernetes.Client
 	}
 	return 0, nil
 }
+
+//ResultUpdate will update the result of pipelines in a table by calling a python script result_update.py
+func ResultUpdate(experimentName string, engineName string, clientSet *chaosClient.LitmuschaosV1alpha1Client) (int, error) {
+
+	//Updating the result table
+	dt := time.Now()
+	app, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
+	Expect(err).To(BeNil(), "Fail to get the chaosresult while updating the result in a table")
+	testVerdict := string(app.Status.ExperimentStatus.Verdict)
+	fmt.Println("The job_id for the job will be", os.Getenv("CI_JOB_ID"))
```

Comment:
Please use some logging package, IF you haven't used yet in this repo, you can do in the next PR

Reply:
yes @chandankumar4 also I will change all pkg errors in the same way